### DISCUSSION TITLE
Nuxt js CORS origin issue

### DISCUSSION THREAD

**Discussion by mahiraltinkaya** [socketio/socket.io#3754] [Q&A] [answered]
Hello there;

I am trying to establish a socket connection between Nuxt js and node js. I've also tried generic solutions for a problem I have never experienced before. I share the configuration below. Can you give information about the error?

![image](https://user-images.githubusercontent.com/25014707/104566734-09216f00-565f-11eb-968d-76692e308fa3.png)

![image](https://user-images.githubusercontent.com/25014707/104566799-1d656c00-565f-11eb-9d50-7f1d0246b0ff.png)

**Comment by darrachequesne** [comment, marked as answer]
Which version of the Socket.IO server are you using?

From the screenshots, I guess you are using Socket.IO client v2 (`?EIO=3`). Please make sure to use the same version on the server and the client.

CORS in v2: https://socket.io/docs/v2/handling-cors/
CORS in v3: https://socket.io/docs/v3/handling-cors/
